examples/entity-aggregation/src/main/java/org/example/ImportCustomersWithRentalsAndPayments.java

```diff
@@ -0,0 +1,71 @@
+package org.example;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.sql.SaveMode;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.functions;
+
+import java.util.Properties;
+
+public class ImportCustomersWithRentalsAndPayments {
+
+    public static void main(String[] args) {
+        // The MarkLogic admin password is assumed to be "admin" per the docker-compose.yml file. This is purely for
+        // demonstrational purposes and should never be used in a real application.
+        final String markLogicAdminPassword = "admin";
+
+        // Create a vanilla local Spark session.
+        SparkSession session = SparkSession.builder()
+            .master("local[*]")
+            .getOrCreate();
+
+        // Define the JDBC properties for connecting to Postgres.
+        Properties jdbcProperties = new Properties();
+        jdbcProperties.setProperty("driver", "org.postgresql.Driver");
+        jdbcProperties.setProperty("user", "postgres");
+        jdbcProperties.setProperty("password", "postgres");
+
+        // See https://stackoverflow.com/a/39129546 for information on aliased queries with Spark. A table name may be
+        // used, but in this example, we want to join customers, rentals, and payments together. The query is placed in
+        // parentheses with an alias after it; the alias can have any name.
+        String query =
```

Comment:
Take a look at `ImportJdbcCommand` in our ETL project - I am thinking that is the better, simpler way to use JDBC with Spark so this complex "aliased query" isn't needed (this was the first approach I found with Spark JDBC so I just went with it), which means the comment can go away too. We'd instead use the "query" option which seems a lot simpler. If you agree, update this and the other example program too.